app/views/digest_mailer/digest_mailer.html.erb

```diff
@@ -13,11 +13,11 @@
             </td>
           </tr>
           <tr>
-          <h1 style="font-family: HelveticaNeue-Bold; font-size: 21px; color: red; line-height: 26px; margin-left: 20px;"> Recent Post </h1><br>
+          <h1 style="font-family: HelveticaNeue-Bold; font-size: 21px; color: red; line-height: 26px; margin-left: 20px;"> Recent Posts </h1><br>
             <% @posts.each do | post | %>
               <h1 style="font-family: HelveticaNeue-Bold; font-size: 21px; color: #331A48; line-height: 26px; margin-left: 20px;"><%= post.title %></h1><br>
               <div style="font-family: HelveticaNeue; font-size: 20px;color: #6E4995; line-height: 18px; padding: 15px;">
-                <p><%= post.body.html_safe %></p><br>
+                <p><%= post.number_of_comments %> comments, <%= post.votes.count %> votes, created at <%= post.created_at.strftime('%Y-%m-%d') %></p><br>
```

Comment:
What if there is just one comment? "1 comments"

Please check this out http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize